### PR TITLE
feature/components

### PR DESCRIPTION
**Pull request by estebanmual**
In this Milestone I:

 - Delete all the boilerplate from CRA
 - Create a React component that matches the given design.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,10 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
```
Comment (AshakaE) [inline src/App.js]: - [x] Kindly change App function to a class based component

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        </tr>
+        <tr>
+          <td colSpan="2"><button type="button">0</button></td>
+          <td><button type="button">.</button></td>
+          <td className={styles.calculator__button__orange}><button type="button">=</button></td>
+        </tr>
+      </tbody>
+    </table>
+  );
+}
```
Comment (AshakaE) [inline src/components/Calculator.js]: - [x] Kindly change Calculator function to a class based component

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @estebanmual , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Git flow ✔️ 
Pull request ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @estebanmual,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**